packages/react/src/CTABanner/CTABanner.tsx

```diff
@@ -0,0 +1,89 @@
+import React, {forwardRef, type Ref} from 'react'
+import clsx from 'clsx'
+import type {BaseProps} from '../component-helpers'
+import styles from './CTABanner.module.css'
+import {Heading as PlainHeading} from '../Heading'
```

Comment:
Recommend we don't rename this to something else entirely.. you can keep as is and prefix the internal version.

https://github.com/primer/brand/pull/128/files#r1025088323